integreat_cms/textlab_api/textlab_api_client.py

```diff
@@ -53,6 +55,9 @@ def benchmark(self, text):
         except URLError as e:
             logger.warning("HIX benchmark API call failed: %r", e)
             return None
+        except OSError as e:
+            logger.warning("HIX benchmark API call failed: %r", e)
+            return None
```

Comment:
```suggestion
        except (URLError, OSError) as e:
            logger.warning(
                "HIX benchmark API call failed with %s: %s", type(e).__name__, e
            )
            return None
```
I think this can be simplified...
However, this does only catch the error in the benchmark call, not in the login call, right? :thinking:
If the API has more general problems, the login might also fail?
Probably, we have to wrap the initialization of the client class itself in a try-except, since the `__init__()` might fail...
https://github.com/digitalfabrik/integreat-cms/blob/03deb9dbeffc32e2c7bbaac8f2b78f4fd99969f5/integreat_cms/cms/views/utils/hix.py#L33-L35

Reply:
Ah that explains why there were further errors. I missed that there were 2 different causes for the problem. Thanks.